packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/service/ServiceRegistrationState.ts

```diff
@@ -192,6 +194,10 @@ export class ServiceConfigState {
     this.TEMPORARY__useStoreModel = val;
   }
 
+  setUseGenerateLineage(val: boolean): void {
+    this.TEMPORARY__useGenerateLineage = val;
```

Comment:
you might need to add setUseGenerateLineage: action as well

Reply:
why have we resolved this comment and not addressed the comment?